santoku/aws/test_sqs_handler.py

```diff
@@ -117,10 +117,7 @@ def test_check_message_attributes_are_well_formed(self, sqs_handler, message_att
         # Message with more than 10 attributes. Failure expected.
         num_attributes = len(message_attributes)
         max_num_attributes = 10
-        message_attribute_content = {
-            "DataType": "String",
-            "StringValue": "Test string value",
-        }
+        message_attribute_content = message_attributes["TestAttribute1"]
```

Comment:
Just realized that this fixture `message_attributes` could have more descriptive keys. As easy as `StringAttribute`, `NumberAttribute` and `BinaryAttribute`.